packages/pagination/demo/OffsetPagination.stories.mdx

```diff
@@ -14,8 +13,18 @@ import README from '../README.md';
 
 <Canvas>
   <Story
-    name="Pagination"
-    args={{ currentPage: 1, totalPages: 11 }}
+    name="OffsetPagination"
+    args={{
+      currentPage: 1,
+      totalPages: 11,
+      labels: {
+        gap: 'Ellipsis indicating non-visible pages',
+        page: p => `Page ${p}`,
+        next: 'Next page',
+        previous: 'Previous page'
+      },
```

Comment:
Not sure, due to the type mix. But can this be setup so it can be controlled via the story?